### DISCUSSION TITLE
Starting the experiment from the previous state as the initial state.

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#4025] [Q&A]
I want to run a new experiment with the initial state of the cell as the state after the previous run experiment. However, the new solution should not contain the data/ plots of the previous run experiment.
Likewise, I tried using sim2.solve(starting_sol=sol1) but the new solution (sol2)also contains the data of previous run experiment (sol1).
Is there any way to extract/get the solution (sol2) only for the new experiment, with the initial state for the new experiment as the final state of the old experiment?

**Comment by valentinsulzer** [comment]
Do `new_model = model.set_initial_conditions_from(solution)` then create a new Simulation with `new_model`
